phoenix-core-client/src/main/java/org/apache/phoenix/compile/CreateIndexCompiler.java

```diff
@@ -161,13 +162,13 @@ private void verifyIndexWhere(ParseNode indexWhere, StatementContext context,
             column =  dataTable.getColumns().get(i);
             value = column.getViewConstant();
             if (value == null) {
-                stringBuilder.append(column.getName().getString() + ",");
```

Comment:
I'm pretty sure whe already have helper functions for this, let's not do this manually.

Reply:
@stoty thanks for pointing it out, I made this change.